pkg/resolver/search_test.go

```diff
@@ -40,7 +44,9 @@ func Test_SearchResolver_Count_WithRBAC(t *testing.T) {
 	// Create a SearchResolver instance with a mock connection pool.
 	val1 := "Pod"
 	searchInput := &model.SearchInput{Filters: []*model.SearchFilter{{Property: "kind", Values: []*string{&val1}}}}
-	resolver, mockPool := newMockSearchResolver(t, searchInput, nil, &ud)
+	PropTypes := make(map[string]string)
+	PropTypes["kind"] = "string"
```

Comment:
Can be simplified with inline declaration.  There's several more places where this can be changed below.
```suggestion
	PropTypes := map[string]string { "kind":"string" }
```